docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -201,8 +207,8 @@ This review step is to ensure that all acceptance tests pass on the release bran
 
         git clone git@github.com:ClusterHQ/flocker.git
         cd flocker
-        git checkout -b *release branch*
-        mkvirtualenv flocker-release-${VERSION}
+        git checkout "release/flocker-${VERSION}"
```

Comment:
:question: The engineer doing the release does

```
$ git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
$ cd flocker-${VERSION}
```

Perhaps it would be better to have these consistent - I guess the reason to clone into a directory called "flocker-${VERSION}" and not just "flocker" is to avoid conflict with the "flocker" directory that the release engineer will likely have.